prompt2model/utils/api_tools.py

```diff
@@ -40,13 +40,20 @@
 class APIAgent:
     """A class for accessing API-based models."""
 
-    def __init__(self, model_name: str = "gpt-3.5-turbo"):
+    def __init__(
+        self,
+        model_name: str = "gpt-3.5-turbo",
+        max_tokens: int | None = None,
+    ):
         """Initialize APIAgent with an API key.
 
         Args:
             model_name: Name fo the model to use (by default, gpt-3.5-turbo).
+            max_tokens: The maximum number of tokens to generate. Defaults to 4x the
```

Comment:
This would move from here in case we decide to move `max_tokens`

Reply:
I didn't 100% understand this comment, but I think you meant the docstring was wrong? In that case fixed.